generate/swaggergen/g_docs.go

```diff
@@ -1187,6 +1187,8 @@ func parseStruct(imports []*ast.ImportSpec, st *ast.StructType, k string, m *swa
 					}
 					if desc := stag.Get("description"); desc != "" {
 						mp.Description = desc
+					} else {
+						mp.Description = strings.ReplaceAll(field.Comment.Text(), "\n", "")
```

Comment:
I am not sure if it's a good idea. I have a concern that this is not a compatible change. And usually we write long message in comments to provide more details, so the `mp.Description` may by too long to show in swagger.

Another concern is that `ReplaceAll`.
For example:
```go
type S struct {
    //some comments look like
   //this setence
   A string 
}
```
But I think only support `tag` to generate swagger is not convenient, using annotation should be allowed.